src/makeRobot.js

```diff
@@ -38,7 +38,68 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    goForward(value = 1) {
+      if (typeof value !== 'number' || value < 1) {
+        return this;
+      }
+      this.coords.y += value;
```

Comment:
You can make it easier
```suggestion
      if (value >= 1) {
         this.coords.y += value;
      }
     
```